python/ray/data/aggregate.py

```diff
@@ -312,3 +368,79 @@ def _to_on_fn(on: Optional[KeyFn]):
         return lambda r: r[on]
     else:
         return on
+
+
+@PublicAPI
+class Quantile(_AggregateOnKeyBase):
+    """Defines Quantile aggregation."""
+
+    def __init__(
+        self,
+        on: Optional[KeyFn] = None,
+        percent: float = 0.5,
```

Comment:
other libraries such as [pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.quantile.html) and [numpy](https://numpy.org/doc/stable/reference/generated/numpy.quantile.html) use the parameter name `q` for quantiles -- should we also use this notation convention?

Reply:
Thank you. I have changed the percentage parameter to q, just like Pandas and Numpy.